ts/features/itwallet/machine/credential/context.ts

```diff
@@ -3,13 +3,15 @@ import { StoredCredential } from "../../common/utils/itwTypesUtils";
 
 export type Context = {
   credentialType: CredentialType | undefined;
+  hardwareKeyTag: string | undefined;
```

Comment:
What's this `hardwareKeyTag` for in this context?

Reply:
I think it's because when we scaffolded this machine we copy-pasted it from the eID issuing one. I removed it, we will surely need to change this context later when we'll introduce the credential issuing logic.